<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Satellite Image Processing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="layerTree">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>841</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="coordinatesLbl">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>660</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Coordinates:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coordinatesTxt">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>660</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="crsLbl">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>660</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CRS:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="crsTxt">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>660</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuRaster">
    <property name="title">
     <string>Raster</string>
    </property>
    <widget class="QMenu" name="menuClassification">
     <property name="title">
      <string>Classification</string>
     </property>
     <addaction name="actionSupervised"/>
     <addaction name="actionUnsupervised"/>
    </widget>
    <addaction name="actionDownload"/>
    <addaction name="actionIndices"/>
    <addaction name="menuClassification"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRaster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../icons/open.png</normaloff>../icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionIndices">
   <property name="text">
    <string>Indices</string>
   </property>
  </action>
  <action name="actionSupervised">
   <property name="text">
    <string>Supervised</string>
   </property>
  </action>
  <action name="actionUnsupervised">
   <property name="text">
    <string>Unsupervised</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
